src/saltext/azurerm/utils/azurerm.py

```diff
@@ -341,10 +343,16 @@ def get_identity_credentials(**kwargs):
                 KnownAuthorities, kwargs.get("cloud_environment", "AZURE_PUBLIC_CLOUD")
             )
         log.debug("AUTHORITY: %s", authority)
+
```

Comment:
**suggestion (non-blocking):** unless a `pre-commit` check like `black` is adding/removing empty lines, it's best to refrain from unrelated whitespace changes in PRs.